### PR TITLE
hack scripts for density testing

### PR DESCRIPTION
**Pull request by spiffxp**
This isn't ready for merge.  Opening early to conform kraken PR tests still pass after the latest kraken-services PR

### CODE REVIEW

**Comment by spiffxp** [comment]
retest this please

**Comment by spiffxp** [comment]
retest this please

**Comment by spiffxp** [comment]
retest this please

**Comment by spiffxp** [comment]
retest this please

**Comment by spiffxp** [comment]
retest this please

**Comment by spiffxp** [comment]
retest this please

**Comment by spiffxp** [comment]
retest this please

**Comment by spiffxp** [comment]
retest this please

**Comment by spiffxp** [comment]
retest this please

**Comment by spiffxp** [comment]
retest this please

**Comment by spiffxp** [comment]
retest this please

**Comment by spiffxp** [comment]
retest this please

File: hack/conformance.sh [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+echo "Conformance test branch: ${KUBE_CONFORMANCE_BRANCH}"
+echo "Conformance test SHA: ${KUBE_CONFORMANCE_SHA}"
+echo "Conformance test cluster size: ${KUBE_CONFORMANCE_NUM_NODES}"
+echo "Conformance test kubeconfig: ${KUBE_CONFORMANCE_KUBECONFIG}"
+
+if $REBUILD_TESTS; then
+  echo
+  # TODO: build just the conformance tests for our platform instead of a cross-platform release?
+  echo "Building conformance tests..."
+  build/make-clean.sh
```
Comment (mikeln) [inline hack/conformance.sh]: current kubernetes build philosophy seems to be use the base Makefile:
make clean
make quick-release

vs calling build scripts directly....

File: hack/density.sh [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+echo "Density test branch: ${KUBE_DENSITY_BRANCH}"
+echo "Density test SHA: ${KUBE_DENSITY_SHA}"
+echo "Density test cluster size: ${KUBE_DENSITY_NUM_NODES}"
+echo "Density test kubeconfig: ${KUBE_DENSITY_KUBECONFIG}"
+
+if $REBUILD_TESTS; then
+  echo
+  # TODO: build just the density tests for our platform instead of a whole release?
+  echo "Building density tests..."
+  build/make-clean.sh
```
Comment (mikeln) [inline hack/density.sh]: Same here: make clean quick-release

**Comment by spiffxp** [comment]
retest this please

**Comment by spiffxp** [comment]
retest this please

**Comment by mikeln** [comment]
LGTM

**Pull request merged**